spec/rspec/rails/example/controller_example_group_spec.rb

```diff
@@ -90,6 +90,30 @@ def my_helper
 
         example.foos_url
       end
+
+      context "when controller is not a stub" do
```

Comment:
This spec seems to fail with the fix as well. Might be some trivial thing is missing.

Reply:
True, it's the Rails version updated :)
 Will fix soon